HelpSource/Guides/Level_Compensation.schelp

```diff
@@ -0,0 +1,158 @@
+title:: Level Compensation when Mixing and Panning
+categories:: Multichannel, Mixing, Panning
+summary:: Balancing Levels when Mixing and Panning in SC
+related:: Classes/Pan2, Classes/PanAz, Classes/Splay, Classes/SplayAz, Classes/LevelComp
+
+section::Considerations
+
+link::Guides/Multichannel-Expansion:: is one of the key features in SuperCollider, making it easy to explore complex polyphonic synthesis and spatialisation. Mixing and panning these signals usually involves going from a number of input channels to a different number of output channels, and for adapting the signal amplitude levels meaningfully, there are two aspects to consider:
+
+strong::
+1. Technical safety: What will be the maximum amplitude of the new signals? In the worst case, can they clip the audio hardware (by going above +-1.0)?::
+
+strong::
+2. Perceived loudness: How loud should the new signals it be, relative to others present?
+::
+
+To balance these aspects, it helps to know more about the signals to be mixed/panned. Most UGens in SC produce a signal amplitude of 1.0, so we will use that value for the discussion here.
+
+strong::Degree of Phase Correlation: :: How likely is it that the momentary peaks of the individual signals will line up in time? The two border cases are fully strong::in-phase: :: the same signal on all channels, so the peaks will always line up, and fully strong::random phase: :: different noise signals on all channels, where the peaks will be statistically very unlikely to ever line up. Most musical signals are somewhere between these extremes, i.e. phase-correlated to some degree.
+
+strong::Equal Amplitude Compensation:: is the safe bet for highly correlated signals: divide the sum of the number of channels; strong::Equal Power Compensation :: is commonly used for weakly correlated signals, as it keeps the acoustic energy constant by dividing the sum through the square root of the number of channels. As most musical signals are somewhere between the extremes, it makes sense to consider and test for the specific case whenever in doubt.
+
+In the acoustics literature, this level compensation factor is called the p-value [1], and it can be generalized as follows:
+
+code::
+
+~levelCompFactor = { |numChans=2, p_value=1|
+	numChans ** p_value.clip(0, 1).neg
+};
+~levelCompFactor.value(2, 1); // equal amplitude, multiply by 1/n
+~levelCompFactor.value(2, 0.5); // equal power, mul by square root of 1/n
+~levelCompFactor.value(2, 0); // keep full level, no level scaling
+::
+
+Here are some examples for the common cases of Mixing and Panning.
+
+subsection:: Mixing M channels to 1
+Mixing down to mono is the simplest case to show the different approaches to level compensation. UGens or functions for this are Mix.ar, sum, mean.
```

Comment:
This is a very nice guide!

One suggestion (optional): Throughout the guide, use the link:: tag when mentioning UGens and code:: when mentioning methods (you could even link to the method if you'd like). We're still working out Help Doc style guidelines, but this is a pretty consistent preference.

So in the above sentence, it would be 
`UGens or functions for this are link::Classes/Mix::, code::sum::, code::mean::.`

Reply:
ok, done!